restangular/restangular.d.ts

```diff
@@ -90,11 +90,20 @@ declare namespace restangular {
     withConfig(configurer: (RestangularProvider: IProvider) => any): IService;
     restangularizeElement(parent: any, element: any, route: string, collection?: any, reqParams?: any): IElement;
     restangularizeCollection(parent: any, element: any, route: string): ICollection;
-    service(route: string, parent?: any): IService;
+    service(route: string, parent?: any): IScopedService;
     stripRestangular(element: any): any;
     extendModel(route: string, extender: (model: IElement) => any): void;
     extendCollection(route: string, extender: (collection: ICollection) => any): void;
   }
+  
+  interface IScopedService extends IService {
+    one(id: number): IElement;
+    one(id: string): IElement;
+    post(elementToPost: any, queryParams?: any, headers?: any): IPromise<any>;
```

Comment:
@andy-ms Yeah, `post()`'s promise isn't guaranteed to resolve with the same type that's posted. For example, the server might only allow certain properties or settings to be posted, then return the full created object.